src/app/static/src/app/components/plots/choropleth/utils.ts

```diff
@@ -12,10 +12,12 @@ export const getFeatureIndicator = function (data: any[],
 
     const result = {} as IndicatorValuesDict;
     iterateDataValues(data, [indicatorMeta], selectedAreaIds, filters, selectedFilterValues,
-        (areaId: string, indicatorMeta: ChoroplethIndicatorMetadata, value: number) => {
+        (areaId: string, indicatorMeta: ChoroplethIndicatorMetadata, value: number, values: any) => {
             result[areaId] = {
                 value: value,
-                color: getColor(value, indicatorMeta, colourRange)
+                color: getColor(value, indicatorMeta, colourRange),
+                lower_value: values['lower']? values['lower']: 0,
```

Comment:
Ah you're setting this here - maybe let this be null or undefined to distinguish from real lower 0 value? Could make lower_value and upper_value optional properties in the type and not set them at all if not required.